shared/haml/onetrust_cookie_scripts.haml

```diff
@@ -1,13 +1,16 @@
 - require '../shared/middleware/helpers/experiments'
 -# OneTrust Cookies Consent Notice scripts for code.org
 - cookie_script_env = experiment_value('onetrust_cookie_scripts')
-- if cookie_script_env == 'production'
-  %script{src: 'https://cdn.cookielaw.org/consent/27cca70a-7db3-4852-9ef0-a6660fd0977d/OtAutoBlock.js', type: 'text/javascript'}
-  %script{src: 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js', type: 'text/javascript', charset: 'UTF-8', 'data-domain-script' => '27cca70a-7db3-4852-9ef0-a6660fd0977d'}
-  :javascript
-    function OptanonWrapper() { }
+- if cookie_script_env == 'off'
+  return
 - elsif cookie_script_env == 'test'
   %script{src: 'https://cdn.cookielaw.org/consent/27cca70a-7db3-4852-9ef0-a6660fd0977d-test/OtAutoBlock.js', type: 'text/javascript'}
   %script{src: 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js', type: 'text/javascript', charset: 'UTF-8', 'data-domain-script' => '27cca70a-7db3-4852-9ef0-a6660fd0977d-test'}
```

Comment:
What is the difference between test and not test?
In other words, why is the test `data domain script` different from production? and Why the development `data-domain-script` will be the same as production?

Reply:
I should have given more details since it is confusing otherwise. The first thing to understand is that there are 6 different versions:
* code.org: dev, test, prod
* onetrust: off, test, prod

The `cookie_script_env` should map to a OneTrust version either "off", "test", or "production" . 
`cookie_script_env` is configured using DCDO. Currently, only the code.org prod servers have the configuration `cookie_script_env=production`. Since code.org dev and test servers don't have the configuration, they default to `cookie_script_env=off`

This PR changes the default `cookie_script_env=off` to instead be `cookie_script_env=production`

> What is the difference between test and not test?

OneTrust has a "test" and "production" version of it's library. By setting the DCDO configuration `cookie_script_env=test`, we can try out the "test" version of the OneTrust libraries.

Reply:
I see, so the `cookie_script_env` doesn't mean we are using a different `data-domain-script` in different environments, it just allow us to use a the two OneTrust libraries changing the flag for all of our environments. 👍